app/controllers/ActivateAccountController.scala

```diff
@@ -0,0 +1,39 @@
+package controllers
+
+import scala.concurrent.{ ExecutionContext, Future }
+
+import com.mohiva.play.silhouette.api._
+import javax.inject.Inject
+import com.mohiva.play.silhouette.api.crypto.CrypterAuthenticatorEncoder
+import com.mohiva.play.silhouette.impl.authenticators.{JWTAuthenticator, JWTAuthenticatorSettings}
+import play.api.i18n.I18nSupport
+import play.api.mvc.{ AbstractController, AnyContent, ControllerComponents, Request }
+
+import models.services.UserService
+import utils.auth.DefaultEnv
+
+class ActivateAccountController @Inject() (
+  cc: ControllerComponents,
+  silhouette: Silhouette[DefaultEnv],
+  userService: UserService,
+  settings: JWTAuthenticatorSettings,
+  authenticatorEncoder: CrypterAuthenticatorEncoder,
+  )(implicit ex: ExecutionContext
+) extends AbstractController(cc) with I18nSupport {
+
+  def activate(token: String) = silhouette.UnsecuredAction.async {
+    JWTAuthenticator.unserialize(token, authenticatorEncoder, settings).toOption match {
```

Comment:
Oh actually.. do we even need to convert to `Option`? Could we just match directly on the `Try`?